### DISCUSSION TITLE
Release 1.2.0 Beta 11

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#987] [General]
## PLUGIN
- Added more values for manual PDC adjustments
- Audio streamer fixes
  - Added missing bounds check that could lead to a crash
  - Fixed buffering for the midi plugin as it was loosing midi data

## SERVER
- Fixed layout mapping for stereo effects (This solves the mono issues, that have been reported in beta 10.)
- Improved the plugin scanner
  - Better error/timeout handling
  - Pick up scan results in case the master process dies during the scan

All downloads are available at https://audiogridder.com/download/.

<hr /><em>This discussion was created from the release <a href='https://github.com/apohl79/audiogridder/releases/tag/release_1_2_0_beta11'>Release 1.2.0 Beta 11</a>.</em>

**Comment by emack999** [comment]
I don't use AAX, but thought I'd let you know that the AAX portion of the Windows installation fails.

On a positive note, the plugin scanner for the server is much improved.

**Reply by apohl79** [reply]
Thx for the info. Will add a note to the release info about the AAX install error. If you see it, press "Remove the read-only attribute and try again" (about 6x) to finish the install.

**Comment by Pabloscotty** [comment]
Thank you Andreas :) I've still got issues. Plugin scanner works perfectly now, most plugins works but Gullfoss is crashing. Beta 8 is still best for my system (Mac 10.14 plugin - Win7 Server) everything works great.

**Reply by apohl79** [reply]
I tried the VST3 version of the gullfoss demo on win10 and macos. Both seem to work fine. Do you have crash reporting enabled on the server? I might be able so see more details about your issue in that case.

**Reply by Pabloscotty** [reply]
I've got something like this. Second screenshot is from beta 8

![Screenshot 2022-08-19 at 12 06 06](https://user-images.githubusercontent.com/80844841/185596676-862a0b95-f7e7-4a48-a51f-db234dd9a093.jpg)
![Screenshot 2022-08-19 at 12 08 48](https://user-images.githubusercontent.com/80844841/185596703-5731958b-caa2-4b04-87c1-789f00d1709c.jpg)

**Comment by CaBuffalo** [comment]
Awesome, thanks, Andeas. :):)

On Thu, Aug 18, 2022 at 9:44 PM Pabloscotty ***@***.***>
wrote:

> Thank you Andreas :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/987#discussioncomment-3428322>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6E6OWJWCHIO6JW323TVZ4GKHANCNFSM566CSQZQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apiatto** [comment]
Unluckily Logic still recognize the plug as a different one than the version beta 9 and it does not work in the old projects.

**Reply by apohl79** [reply]
This is because I changed audio unit type for the FX plugin, so it can receive midi data. Can you try the following:

1) start logic and open the plugin manager
2) press "Full Audio Unit Reset"
3) terminate logic
4) open the terminal app
5) enter ```rm -rf ~/Library/Caches/AudioUnitCache/``` and press return
6) install version 1.1.1 of the AG plugins
7) reboot
8) start logic (your projects should load now with v1.1)
9) terminate logic
10) install 1.2 beta11
11) start logic (your projects should load with the new ag version now)

**Reply by apiatto** [reply]
Hello, I tried and it worked but with distorted sounds. After a reboot the issue is still the same..
